.github/workflows/botonic-react-tests.yml

```diff
@@ -15,35 +15,35 @@ jobs:
     env:
       PACKAGE: botonic-react
     steps:
-    - name: Checking out to current branch
-      uses: actions/checkout@v2
-    - name: Setting up node
-      uses: actions/setup-node@v2-beta
-      with:
-        node-version: '12'
-    - name: Setting up cache
-      uses: actions/cache@v2
-      with:
-        path: ~/.npm
-        key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
-        restore-keys: |
-          ${{ runner.os }}-node-
-    - name: Install dev dependencies
-      run: (cd ./packages/$PACKAGE && npm install -D)
-    - name: Build
-      run: (cd ./packages/$PACKAGE && npm run build)
-    - name: Run tests
-      run: (cd ./packages/$PACKAGE && npm run test)
-    - name: Verify lint
-      run: (cd ./packages/$PACKAGE && npm run lint_ci)
-    - name: Verify index.d.ts
-      run: scripts/qa/lint-d-ts.sh ./packages/$PACKAGE
+      - name: Checking out to current branch
+        uses: actions/checkout@v2
+      - name: Setting up node
+        uses: actions/setup-node@v2-beta
+        with:
+          node-version: '14'
```

Comment:
This solution of updating each version individually should be temporal, in the future I would like to create a reusable workflow where we just change for 1 file and all workflows update with it

Reply:
yes, we will keep improving it 😉